.github/workflows/cla.yml

```diff
@@ -1,26 +1,37 @@
-name: 'CLA Assistant'
+name: "CLA Assistant"
 on:
   issue_comment:
     types: [ created ]
   pull_request_target:
-    types: [ opened, closed, synchronize ]
+    types: [ opened,closed,synchronize ]
 
 jobs:
   CLAssistant:
     runs-on: ubuntu-latest
     steps:
-      - name: 'CLA Assistant'
-        if: (github.event.comment.body == 'recheckcla' || github.event.comment.body == 'I have read the CLA Document and I hereby sign the CLA') || github.event_name == 'pull_request_target'
-        # Alpha Release
-        uses: gnosis/cla-github-action@master
-        # GitHub token, automatically provided to the action
-        # (No need to define this secret in the repo settings)
+      - name: "CLA Assistant"
+        if: (github.event.comment.body == 'recheck' || github.event.comment.body == 'I have read the CLA Document and I hereby sign the CLA') || github.event_name == 'pull_request_target'
+        # Beta Release
+        uses: cla-assistant/github-action@v2.1.3-beta
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          # the below token should have repo scope and must be manually added by you in the repository's secret
+          PERSONAL_ACCESS_TOKEN: ${{ secrets.PERSONAL_ACCESS_TOKEN }}
```

Comment:
@jpalvarezl We need to set this one up: https://github.com/cla-assistant/github-action#environmental-variables

Reply:
Added to repo secrets